Adam add videos (#692)
returntocorp/semgrep-docs: docs/experiments/taint-propagators.md

Taint propagators allow you to customize how taint is propagated. 

> The following video can help you to understand taint propagators:

Author: ```suggestion
The following video provides a quick overview of taint propagators:
```
